### DISCUSSION TITLE
Temporality for Place

### DISCUSSION THREAD

**Discussion by hekl** [schemaorg/schemaorg#2976] [Ideas]
There is a startDate for Events, a foundingDate for Organisations, a temporalCoverage for creativeWorks, but what about a beginning, end or temporalCoverage for Place and especially its subtypes like AdministrativeArea or  LandmarksOrHistoricalBuildings? The current domain typing for the three mentioned doesn’t allow us to use these on places. I would like to give my places a start or end date or temporal coverage.

**Comment by HughP** [comment]
Can you use additional type?

On Sun, Oct 31, 2021 at 4:10 PM Henk Laloli ***@***.***>
wrote:

> There is a startDate for Events, a foundingDate for Organisations, a
> temporalCoverage for creativeWorks, but what about a beginning, end or
> temporalCoverage for Place and especially its subtypes like
> AdministrativeArea or LandmarksOrHistoricalBuildings? The current domain
> typing for the three mentioned doesn’t allow us to use these on places. I
> would like to give my places a start or end date or temporal coverage.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/schemaorg/schemaorg/discussions/2976>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAJ2JR3MIERAMJMMNV2SPDUJXEGVANCNFSM5HC2BDZA>
> .
>
-- 
All the best,
-Hugh

Sent from my iPhone

**Reply by hekl** [reply]
I don't know how that works. I would extend temporalCoverage to Places.
